src/bindings/python/constraints.txt

```diff
@@ -19,4 +19,4 @@ paddlepaddle==2.4.2
 tensorflow>=1.15.5,<2.14.0
 six~=1.16.0
 protobuf>=3.18.1,<4.0.0
-onnx==1.13.1
+onnx==1.14.0
```

Comment:
Yes, you are right, but I want to see difference between 1.14.0 and 1.14.1. Regarding their release notes - we may expect bugfix. If it won't provide big difference - we will move to 1.14.1 instead of 1.14.0.